Tuto-GUDHI-persistent-entropy.ipynb

```diff
@@ -0,0 +1,823 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        rips.persistence()
</pre><p>if we don't use the returned list, just <code>rips.compute_persistence()</code>  is nicer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>You are right.</p><p>Done!</p><p> </p>